src/app/components/AppLayout/AppLayout.tsx

```diff
@@ -91,7 +92,7 @@ export const AppLayout: FunctionComponent<AppLayoutProps> = ({ children }) => {
       sidebar={Sidebar}
       onPageResize={onPageResize}
     >
-      {children}
+      <ErrorWithDetailBoundary>{children}</ErrorWithDetailBoundary>
```

Comment:
We use the `AppLayout` component only in the "development" versions of the app. You'll have to include the error boundary inside `AppFederated` too. Or you can move it in one component that is included everywhere, like the `Routes` component or something similar.